pom.xml

```diff
@@ -453,33 +437,26 @@
                 <artifactId>maven-source-plugin</artifactId>
             </plugin>
             <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>codenarc-maven-plugin</artifactId>
-            </plugin>
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>findbugs-maven-plugin</artifactId>
+                <groupId>com.github.spotbugs</groupId>
+                <artifactId>spotbugs-maven-plugin</artifactId>
             </plugin>
         </plugins>
     </build>
 
     <profiles>
-        <!-- support Java 8: use newer FindBugs plugin, switch off Javadoc's doclint -->
+        <!-- switch off Javadoc's doclint -->
         <profile>
             <id>jdk8</id>
```

Comment:
I'd get rid of this profile and fold it into the main config.

Reply:
good point, it's leftover from times I thought that it would work with older JDKs. I'll move it to main config.